src/components/FetchError.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const FetchError = ({ errorMessage }) => (
+  <div>{errorMessage}</div>
+);
+
+FetchError.propTypes = {
+  errorMessage: PropTypes.string.isRequired
```

Comment:
Please use tailing commas. Otherwise adding line will be detected by git as changing 2 lines